opsdroid/loader.py

```diff
@@ -352,19 +357,18 @@ def include_constructor(loader, node):
             included_yaml = os.path.join(main_yaml_path, loader.construct_scalar(node))
 
             with open(included_yaml, "r") as included:
-                return yaml.safe_load(included)
-
-        yaml.SafeLoader.add_constructor("!envvar", envvar_constructor)
-        yaml.SafeLoader.add_constructor("!include", include_constructor)
+                return yaml.load(stream, Loader=cls.loader)
```

Comment:
Here you need to load the included file not the stream, please replace this with:
`return yaml.load(included, Loader=cls.yaml_loader)`